File: Code 3 Callouts/Code 3 Callouts/Util/Logger.vb
Imports Rage

Namespace Util

    Friend Module Logger

        Friend Sub LogTrivial(ByVal pMessage As String)
            Try
                Game.LogTrivial(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogVerbose(ByVal pMessage As String)
            Try
                Game.LogVerbose(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogVeryVerbose(ByVal pMessage As String)
            Try
                Game.LogVeryVerbose(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogExtremelyVerbose(ByVal pMessage As String)
            Try
                Game.LogExtremelyVerbose(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogTrivialDebug(ByVal pMessage As String)
            Try
                Game.LogTrivialDebug(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogVerboseDebug(ByVal pMessage As String)
            Try
                Game.LogVerboseDebug(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Friend Sub LogExtremelyVerboseDebug(ByVal pMessage As String)
            Try
                Game.LogExtremelyVerboseDebug(FormatMessage(pMessage))
            Catch ex As Exception
            End Try
        End Sub

        Private Function FormatMessage(ByVal pMessage As String) As String
            Return String.Format("[{0}] {1}", My.Application.Info.Title, pMessage)
        End Function

    End Module

End Namespace

File: Code 3 Callouts/Code 3 Callouts/Util/Logger.cs
using Rage;
using System;

namespace Stealth.Plugins.Code3Callouts.Util
{

	internal static class Logger
	{

		internal static void LogTrivial(string pMessage)
		{
			try {
				Game.LogTrivial(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogVerbose(string pMessage)
		{
			try {
				Game.LogVerbose(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogVeryVerbose(string pMessage)
		{
			try {
				Game.LogVeryVerbose(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogExtremelyVerbose(string pMessage)
		{
			try {
				Game.LogExtremelyVerbose(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogTrivialDebug(string pMessage)
		{
			try {
				Game.LogTrivialDebug(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogVerboseDebug(string pMessage)
		{
			try {
				Game.LogVerboseDebug(FormatMessage(pMessage));
			} catch {
			}
		}

		internal static void LogExtremelyVerboseDebug(string pMessage)
		{
			try {
				Game.LogExtremelyVerboseDebug(FormatMessage(pMessage));
			} catch {
			}
		}

		private static string FormatMessage(string pMessage)
		{
			return string.Format("[{0}] {1}", Common.VersionInfo.ProductName, pMessage);
		}

	}

}